types/src/legacy_sse_data/translate_execution_result.rs

```diff
@@ -163,8 +160,11 @@ fn maybe_tanslate_stored_value(stored_value: &StoredValue) -> Option<TransformKi
         StoredValue::ByteCode(_) => None,
         StoredValue::MessageTopic(_) => None,
         StoredValue::Message(_) => None,
-        StoredValue::Reservation(_) => None,
+        StoredValue::Prepaid(_) => None,
         StoredValue::EntryPoint(_) => None,
+        StoredValue::LegacyTransfer(transfer_v1) => {
```

Comment:
There is currently a bug in casper-node/types. This should be named StoredValue::Types, we will need to update this once it gets fixed in node code